mlops/parallelm/mlops/metrics_constants.py

```diff
@@ -4,18 +4,21 @@
 class ClassificationMetrics(Enum):
     """
     Class will hold predefined naming of all classification ML Metrics supported by ParallelM.
+    When add new enum, please register stat creation function to ClassificationStatObjectFactory.registry_name_to_function.
     """
     ACCURACY_SCORE = "Accuracy Score"
     AUC = "AUC(Area Under the Curve)"
     AVERAGE_PRECISION_SCORE = "Average Precision Score"
     BALANCED_ACCURACY_SCORE = "Balanced Accuracy Score"
     BRIER_SCORE_LOSS = "Brier Score Loss"
+    CLASSIFICATION_REPORT = "Classification Report"
```

Comment:
maybe you can iterate over enums and registered handlers and check if their number match

Reply:
i added that thing to my last pr: https://github.com/mlpiper/mlpiper/pull/95